grpcclient/grpcclient.go

```diff
@@ -118,6 +129,14 @@ func (cfg *Config) DialOption(unaryClientInterceptors []grpc.UnaryClientIntercep
 		)
 	}
 
+	if cfg.InitialStreamWindowSize > defaultInitialWindowSize {
```

Comment:
[nit] I would add a comment to explain **why** (BDP estimator being turned off when it's explicitely set, even at the same value of the default one).

Reply:
Good point, added in 003f044.